src/log/Log.cc

```diff
@@ -354,9 +354,8 @@ void Log::_flush(EntryQueue *t, EntryQueue *requeue, bool crash)
 void Log::_log_message(const char *s, bool crash)
 {
   if (m_fd >= 0) {
-    int r = safe_write(m_fd, s, strlen(s));
-    if (r >= 0)
-      r = safe_write(m_fd, "\n", 1);
+    std::string b {s, '\n'};
```

Comment:
Ya, you're right. I've sat on this branch for too long and forgot to double-check that was correct...